src/traffic_cop/traffic_cop.cpp

```diff
@@ -364,7 +364,7 @@ TrafficCopResult TrafficCop::ExecuteExplainStatement(
   out->WriteDataRow(reinterpret_cast<const byte *const>(&plan_string_val), output_columns,
                     {network::FieldFormat::text});
 
-  return {ResultType::COMPLETE, 0};
+  return {ResultType::COMPLETE, static_cast<uint32_t>(0)};
```

Comment:
Hate to come back with this on a one-line diff, but perhaps just use an integer literal here to be a bit more concise?

```
return {ResultType::COMPLETE, 0U};
```

It looks like this is the approach we take elsewhere in the `TrafficCop`.